pkgs/development/python-modules/gunicorn/default.nix

```diff
@@ -8,23 +8,18 @@
 
 buildPythonPackage rec {
   pname = "gunicorn";
-  version = "20.0.4";
-  disabled = isPy27;
+  version = "20.1.0";
+  disabled = pythonOlder "3.5";
```

Comment:
I have to imagine that disabling `gunicorn` on older python versions might be a bit controversial upstream. They may ask you to make an 20.0.X variant.  I think this is fine for us, though.